lld/COFF/Options.td

```diff
@@ -299,6 +299,8 @@ def : Flag<["--"], "time-trace">, Alias<time_trace_eq>,
 def time_trace_granularity_eq: Joined<["--"], "time-trace-granularity=">,
     HelpText<"Minimum time granularity (in microseconds) traced by time profiler">;
 
+defm build_id: B<"build-id", "Generate build ID", "Do not Generate build ID">;
```

Comment:
Should we mention, here or in a doc somewhere, that this is always on when generating a PDB?